.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+This PR closes #{issue_number}.
+
+<!-- 
+    Write a short description to explain changes that are not mentioned in the initial issue.
+    What were the reasons for those changes?
+    Which decisions did you make and why? 
+    What else should reviewers know about your changes? 
+-->
+
+## Readiness checklist
+
+<!-- 
+    If you want your changes to be merged quickly,
```

Comment:
In one follows CONTRIBUTING, no guarantee at all that PR will be merged quickly. 
Suggestion to change it to smth like:

> Please follow contributing guidelines to meet PR requirements and decrease the change request count.